### DISCUSSION TITLE
Can I take consecutive frames from RTSP stream?

### DISCUSSION THREAD

**Discussion by Simopich** [ultralytics/yolov5#8358] [Q&A] [answered]
Hi, I was wondering if it was already possible to take a specified number of consecutive frames from an RTSP stream before proceeding to the detection (Ignoring the new frames lost in the detection interval).

Thanks in advance!

**Comment by glenn-jocher** [comment, marked as answer]
@Simopich `read` variable on L349 controls this
https://github.com/ultralytics/yolov5/blob/34df5032a7d2e83fe3d16770a03bd129b115d184/utils/dataloaders.py#L347-L363

**Reply by Simopich** [reply]
Thank you!
